public/lab-files/Lesson02/Challenges/images.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Images</title>
+</head>
+<body>
+    <h1>The perfect trip</h1>
+    <p>I would go to...</p>
+    <img src="http://www.worldatlas.com/webimage/countrys/oceania/lgcolor/nzcolor.gif" width="499"/>
```

Comment:
When you have an image tag, you need to add the attribute alt with a description of what the image is.